app/src/main/java/com/quran/labs/androidquran/util/QuranFileUtils.java

```diff
@@ -150,10 +148,7 @@ public boolean haveAllImages(Context context, String widthParam, int totalPages)
         return true;
       } else {
         Timber.d("haveAllImages: couldn't find the directory, so making it instead");
-        makeQuranDirectory(context);
-        if (!IMAGES_DIRECTORY.isEmpty()) {
-          makeQuranImagesDirectory(context);
-        }
+        makeQuranDirectory(context, widthParam);
```

Comment:
this is equivalent - with one exception that we're explicitly passing the width in instead of using the default width.

Reply:
should revisit this though because when coming from the code to check fallbacks, it'll end up creating at least 3 directories, most of which aren't needed